### PR TITLE
feat(Slider): new component

### PR DESCRIPTION
**Pull request by lisalupi**
## Type

- Feature

#### What is expected?

New component `<Slider />`

### CODE REVIEW

File: packages/ui/src/components/Slider/SingleSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+
+  const tooltipText = useMemo(() => {
+    if (tooltip === true) {
+      return computedValue
+    }
+    if (tooltip) {
+      return tooltip
+    }
+
+    return null
```
Comment (matthprost) [inline packages/ui/src/components/Slider/SingleSlider.tsx:244]:
```suggestion
    return undefined
```

File: packages/ui/src/components/Slider/SingleSlider.tsx [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+            {required ? (
+              <Icon name="asterisk" sentiment="danger" size={8} />
+            ) : null}
+          </Stack>
+
+          {direction === 'column'
+            ? styledValue(customValue ?? valueToShow)
+            : null}
+        </Stack>
+      ) : null}
```
Comment (matthprost) [inline packages/ui/src/components/Slider/SingleSlider.tsx:274]: label is same in both single and double, couldn't it be unified in a`Label.tsx` component?

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by matthprost — CHANGES_REQUESTED** [review]
Just noticed also the options label under seems not aligned with the slider

Screenshot 2024-07-01 at 16 07 00

File: packages/ui/src/components/Slider/DoubleSlider.tsx [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+
+const SliderElement = styled('input', {
+  shouldForwardProp: prop => !['themeSlider', 'suffix'].includes(prop),
+})<{ themeSlider: string; disabled: boolean; suffix: boolean }>`
+  position: absolute;
+  width: 100%;
+  pointer-events: none;
+  appearance: none;
+  height: 100%;
+  opacity: 1;
```
[the remark this answers is not in the document]
Comment (lisalupi) [reply packages/ui/src/components/Slider/DoubleSlider.tsx:33]: no, forgot to delete that ^^

File: packages/ui/src/components/Slider/DoubleSlider.tsx [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+  onChange,
+  min = 0,
+  max = 100,
+  step = 1,
+  id,
+  onBlur,
+  onFocus,
+  possibleValues,
+  options,
+  optionsUnit,
```
[the remark this answers is not in the document]
Comment (lisalupi) [reply packages/ui/src/components/Slider/DoubleSlider.tsx:131]: well it shouldn't be different, i can merge both

**Review by PetmePant — APPROVED** [review]

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by matthprost — APPROVED** [review]

**Review by radhi-nasser-scaleway — APPROVED** [review]

**Review by ValentinRLL — APPROVED** [review]

**Pull request merged**